tests/testthat/test-surv-brier.R

```diff
@@ -0,0 +1,49 @@
+test_that('case weights', {
+  lung_surv <- data_lung_surv() %>% dplyr::filter(.time == 100)
+  lung_surv$case_wts <- rep(2, nrow(lung_surv))
+
+  brier_res <- brier_survival(
+    data = lung_surv,
+    truth = surv_obj,
+    estimate = .pred_survival,
+    censoring_weights = ipcw,
+    eval_time = .time
+  )
+
+  brier_res_case_wts <- brier_survival(
+    data = lung_surv,
+    truth =  surv_obj,
+    estimate = .pred_survival,
+    censoring_weights = ipcw,
+    case_weights = case_wts,
+    eval_time = .time
+  )
+
+  expect_equal(
+    brier_res$.estimate,
+    brier_res_case_wts$.estimate
+  )
+})
+
+# sklearn compare --------------------------------------------------------------
+
+test_that('sklearn equivalent', {
+  py_res <- read_pydata("py-brier-survival")
+
+  eval_times <- c(100, 500, 1000)
+
+  lung_surv <- data_lung_surv()
```

Comment:
An out-of-scope comment:
I don't quite understand what the purpose of that function is. Is that so that the dataset only exists within the environment of that call to `test_that()`? 

If so, you could use the `envir` argument to `data` à la
```
data(time_to_million, package = "censored", envir = rlang::current_env())
```
and put that into the `test_that()` call directly.

Reply:
That is a good point. I am currently following the existing codebase where these functions do more complicated things. I'll open an issue to work on that globally